apl-core/src/test/java/com/apollocurrency/aplwallet/apl/exchange/dao/DexTradeDaoTest.java

```diff
@@ -98,6 +98,70 @@ void testInsert() {
                         
         assert(extractedTradeEntries.size() == (iters + 1));
         
+        for (Integer i=0; i<extractedTradeEntries.size(); i++) {           
+            DexTradeEntry current = extractedTradeEntries.get(i);
+            DexTradeEntry stored  = storedEntries.get(i);
+            // comparing current 
+            assert ( current.getSenderOfferID() == stored.getSenderOfferID() );
```

Comment:
Do not use assert keyword for tests: https://www.baeldung.com/java-assert#best-practices